docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,136 @@
+.. warning::
+    This is alpha functionality. While usable, this has not yet been tested in a large
+    number of different scenarios. The API may also still be subject to change.
+
+Piezo tracking
+==============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+In this tutorial, we will determine a high frequency distance (piezo distance) between the beads from the piezo mirror position of trap 1 and the measured force data.
+We will also show how to use the same reference curve to apply baseline correction for force signals in post processing.
+
+Positional Calibration
+----------------------
+
+The first step is to perform a positional calibration to calibrate the trap position to the distance measured by the camera.
+
+To do this calibration, we require a dataset acquired in the absence of a tether.
+In this dataset, trap 1 is moved over range, while the trap 1 position and camera-based distance is recorded.
+
+Note that this dataset should be acquired such that the bead tracking templates do not overlap.
+In case of overlap, make sure to slice the position and distance data such that this region is not included.
+Let's load this dataset and perform the distance calibration by invoking::
+
+    no_tether_data = lk.File("no_tether.h5")
+
+    distance_calibration = lk.DistanceCalibration(
+        calibration_data["Trap position"]["1X"], calibration_data.distance1, degree=2
+    )
+
+What this class does is regress a polynomial between the trap position and distance determined by the bead tracking.
+Note that this bead-to-bead distance typically already has the bead diameter subtracted, therefore it reflects the surface-to-surface distance.
+We can plot what this curve looks like by invoking `.plot()` on it::
+
+    distance_calibration.plot()
+
+.. image:: figures/piezotracking/mirror_calibration.png
+
+We can also inspect the residual, to determine how well the calibration model describes the data::
+
+    distance_calibration.plot_residual()
+
+.. image:: figures/piezotracking/mirror_calibration_residual.png
```

Comment:
oof, this is a rough residual plot. Is it typical to see this kind of structure in the residuals? If so, we should probably mention something about it because it's very very apparent that this is not a particularly great fit

Reply:
Yeah, I agree. I think we should find a better dataset. I suspect something may have gone wrong with the template selection here. The template error for bead 1 also seems to depend on the mirror position.

Reply:
So looking at a few recent datasets, this residual magnitude is pretty typical. It's about 15 nm peak to peak; so that's about 45 base pairs. For pulling this is probably not super relevant (most constructs are much bigger and even the error in the um per kbp value is larger) and other sources of errors are dominant.

For folding/unfolding studies, the distance transitions are mostly gonna be dominated by the force part (since one doesn't move the mirror when looking at those).

I do think that is warrants some additional investigation, but for now, this is just what it is, and I still think that showing how to spin up the residual plot is useful.

I've added the following:
```
We can see that there is some error (the discrepancy does not scatter randomly around zero), but for this experiment, 15 nm error is within an acceptable range::
```

To explicitly note that this is a judgment call that needs to be made and not gloss over it. What do you think?